mtp_noms_ops/apps/security/forms.py

```diff
@@ -232,10 +232,8 @@ def __init__(self, request, **kwargs):
                                                            title=_('All prisons'))
         self['prison_region'].field.choices = insert_blank_option(prison_details_choices['regions'],
                                                                   title=_('All regions'))
-        self['prison_population'].field.choices = insert_blank_option(prison_details_choices['populations'],
-                                                                      title=_('All types'))
-        self['prison_category'].field.choices = insert_blank_option(prison_details_choices['categories'],
-                                                                    title=_('All categories'))
+        self['prison_population'].field.choices = prison_details_choices['populations']
```

Comment:
i think population still needs `insert_blank_option` else you'll always be filtering on this